client/src/components/SidePanel/Agents/AgentConfig.tsx

```diff
@@ -225,15 +225,14 @@ export default function AgentConfig({
             name="description"
             control={control}
             render={({ field }) => (
-              <input
+              <textarea
```

Comment:
A selfclosing TextArea is ONLY valid in XHTML (exclusively) even though it is processed just fine by some browsers.
This should be changed to the standard OpenTag & CloseTag format.